apps/passport-server/src/database/queries/e2ee.ts

```diff
@@ -64,10 +65,10 @@ export async function updateEncryptedStorage(
       ($1, $2) ON CONFLICT(blob_key) DO UPDATE SET encrypted_blob = $1`,
       [newBlobKey, encryptedBlob]
     );
-    await txClient.query("UPDATE users SET salt = $2 WHERE uuid = $1", [
-      uuid,
-      newSalt
-    ]);
+    await txClient.query(
```

Comment:
This will conflict with my outstanding PR #803, so whichever of us merges second will have to deal.  There shouldn't be any difficult conflict to handle.  Adding the new `encryption_key = NULL` to this line in my version of this function is safe, since it doesn't interact with the new revision-tracking functionality.